cluster/config-defaults.yaml

```diff
@@ -342,10 +342,10 @@ skipper_open_policy_agent_styra_token: ""
 #   - disabled: scales controller to zero replicas
 #   - production: runs the controller
 #
-fabric_gateway_controller_mode: "disabled"
+fabric_gateway_controller_mode: "production"
 fabric_gateway_controller_cpu: "50m"
 fabric_gateway_controller_memory: "150Mi"
-fabric_gateway_crd_v1_enabled: "false"
+fabric_gateway_crd_v1_enabled: "true"
```

Comment:
Let's drop all config items that are only to "enable" it.
https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/fabric-gateway/fabricgateway_crd.yaml#L1